src/utils/contractHelpers.ts

```diff
@@ -71,6 +72,10 @@ export const getSouschefContract = (id: number, web3?: Web3) => {
   const abi = config.poolCategory === PoolCategory.BINANCE ? sousChefBnb : sousChef
   return getContract(abi, getAddress(config.contractAddress), web3)
 }
+export const getSouschefV2Contract = (id: number, web3?: Web3) => {
```

Comment:
I don't really like littering the codebase with comments... but do you think it's worth adding something here explaining that this contract is only needed for the pools tests when calling the `stakedToken` method?